x-pack/filebeat/module/kubernetes/_meta/fields.yml

```diff
@@ -0,0 +1,134 @@
+- key: kubernetes
+  title: Kubernetes
+  description: >
+    Module for handling logs produced by Kubenetes pods
+  fields:
+  - name: coredns
+    type: group
+    description: >
+      Fields from coredns logs after normalization
+    fields:
+    - name: duration
```

Comment:
Please add descriptions for fields in this file.

Reply:
Check out how I handled `event.duration` across very different sources in [#10274](https://github.com/elastic/beats/pull/10274/files#diff-dd78bfc56f97616f4fd4cd4a90b832ed). I think that's a pattern we can likely reuse broadly. This will reduce compilations a lot across the board, if we can always use the same snippet.